cfme/tests/cli/test_appliance_console.py

```diff
@@ -1409,14 +1409,10 @@ def test_appliance_console_negative():
     pass
 
 
-@pytest.mark.manual
 @pytest.mark.tier(2)
-@pytest.mark.meta(coverage=[1753687])
```

Comment:
Bug 1753687 is closed so removed this marker

Reply:
please add @pytest.mark.meta(automates=[BZ(1753687)]) as this BZ is now automated.